common/app/experiments/Experiments.scala

```diff
@@ -23,22 +22,13 @@ object ActiveExperiments extends ExperimentsDefinition {
 
 object UseSourcepointPropertyId
     extends Experiment(
-      name = "use-sourcepoint-property-id",
+      name = "use-sourcepoint-property-idDSTEST",
```

Comment:
```suggestion
      name = "use-sourcepoint-property-id",
```

There should be no diff here

Reply:
Oh no one of my local testing strategies - thanks Emma